.github/workflows/release-dataset.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish dataset to production DB
+
+on:
+  workflow_dispatch:
+    branches: 
+      - main
```

Comment:
Who is allowed to run this?  Is it anyone with access to the repo?  The production one should be a bit more exclusive maybe?  Just wondering.

Reply:
Thanks for asking! Right now anyone with write access to this repo can run it, I'll talk to Megan to see if we should set up different access rules for releasing datasets to production.